spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskDefinitionController.java

```diff
@@ -219,10 +223,26 @@ public TaskDefinitionResource display(@PathVariable("name") String name) {
 	 */
 	class Assembler extends RepresentationModelAssemblerSupport<TaskExecutionAwareTaskDefinition, TaskDefinitionResource> {
 
+		private boolean enableManifest = false;
```

Comment:
Technically we don't have to initialize a boolean at the class level to false.  I'll fix on merge.